InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java

```diff
@@ -50,59 +50,21 @@ public void initialize() {
 
 
 
-        ds = DriverStation.getInstance();
+        driverStation = DriverStation.getInstance();
 
+        leftMotors = new WPI_TalonSRX[config.drive.MOTORS_PER_SIDE];
+        rightMotors = new WPI_TalonSRX[config.drive.MOTORS_PER_SIDE];
 
-
-        // left and right leader motor device IDs
-        int leftLeader = DriveConstants.LEFT_MOTOR_IDS[0];
-        int rightLeader = DriveConstants.RIGHT_MOTOR_IDS[0];
-
-        leftMotors = new WPI_TalonSRX[DriveConstants.MOTORS_PER_SIDE];
-        rightMotors = new WPI_TalonSRX[DriveConstants.MOTORS_PER_SIDE];
-        
-        // initialize all motors
-        for (int i = 0; i < DriveConstants.MOTORS_PER_SIDE; i++) {
-            leftMotors[i] = new WPI_TalonSRX(DriveConstants.LEFT_MOTOR_IDS[i]);
-            rightMotors[i] = new WPI_TalonSRX(DriveConstants.RIGHT_MOTOR_IDS[i]);
-
-            // reset to factory defaults
-            MotorUtils.initializeMotorDefaults(leftMotors[i]);
-            MotorUtils.initializeMotorDefaults(rightMotors[i]);
-
-            // set follower to corresponding leader if not already the leader
-            if (i != 0) {
-                leftMotors[i].set(ControlMode.Follower, leftLeader);
-                rightMotors[i].set(ControlMode.Follower, rightLeader);
-            }
+        for (int i = 0; i < config.drive.MOTORS_PER_SIDE; i++) {
+            leftMotors[i] = MotorUtils.makeSRX(config.drive.leftMotors[i]);
+            rightMotors[i] = MotorUtils.makeSRX(config.drive.rightMotors[i]);
         }
 
-
-
-        // initialize PID for leaders
-
-        MotorUtils.initializeMotorFPID(
-            leftMotors[0],
-            DriveConstants.LEFT_VEL_KF,
```

Comment:
We should remove any constants we aren't using in DriveContants so they don't confuse people.